Abcrete/BangazonAPI
# I fixed the issue in training program table, now you can only delete …

## Status

Choose:
**IN DEVELOPMENT**

## Migrations
NO

## Description
A few sentences describing the overall goals of the pull request's commits.
I fixed the issue in training program table, now you can only delete it if it has not started, also created model for employeeTraining and uncommented the code in the context (dbSet)

## Todos
Make sure it creates a table for employeeTraining in the database 


## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1.
```
git checkout az-test
```
2. delete the migrations file and database in your project and 
3.
```
dotnet ef migrations add Initial
dotnet ef update database
```
4. open the database in your sql app and you should see the table for employeeTraining
5. Add some data in the TrainingProgram table and make sure startDate is in the past, now you will be able to delete that trainingProgram

## Impacted Areas in Application
List general components of the application that this PR will affect:
TrainingProgramController.cs, BangazonContext.cs, Employee.cs, EmployeeTraining.cs



## Link to Feature Ticket
#64
